assets/js/modules/ads/datastore/module-data.test.js

```diff
@@ -24,13 +24,24 @@ import {
 	untilResolved,
 } from '../../../../../tests/js/utils';
 import { initialState } from './index';
-import { MODULES_ADS } from './constants';
+import { MODULES_ADS, PLUGINS } from './constants';
 
 describe( 'modules/ads module data', () => {
 	const baseModulesGlobalName = '_googlesitekitModulesData';
 	const baseData = {
 		ads: {
 			supportedConversionEvents: [ 'add-to-cart' ],
+			plugins: {
+				woocommerce: {
+					active: false,
+					installed: false,
+				},
+				'google-listings-and-ads': {
```

Comment:
Maybe we should use:
```suggestion
				[ PLUGINS.GOOGLE_FOR_WOOCOMMERCE ]: {
```
here too? 🤔